src/scenes/Activities/ActivityHistory/EventRows/index.js

```diff
@@ -33,33 +33,50 @@ class EventRow extends Component {
       this.setState({ expanded: !this.state.expanded });
     }
 
-    onEventNameClick = (topicAddress) => async (event) => {
+    onEventNameClick = (eventAddress) => async (event) => {
       event.stopPropagation();
-      if (topicAddress) {
-        const { activities: { history: { getOracleAddress } } } = this.props.store;
+      if (eventAddress) {
+        const { activities: { history: { getEventAddress } } } = this.props.store;
         const { history } = this.props;
-        const nextLocation = await getOracleAddress(topicAddress);
+        const nextLocation = await getEventAddress(eventAddress);
         if (nextLocation) history.push(nextLocation);
       }
     }
 
     render() {
       const { transaction, intl, classes } = this.props;
-      const { name, topic, type, txid, amount, token, fee, status, createdTime } = transaction;
+      const { txType, txid, txReceipt: { cumulativeGasUsed }, txStatus } = transaction;
       const { expanded } = this.state;
 
+      // parse necessary data
+      let name;
+      let address;
+      let amount;
+      if (txType === TransactionType.CREATE_EVENT) {
+        name = transaction.name; // eslint-disable-line
```

Comment:
destructure the transaction fields. only disable eslint if necessary. the rules are there for a reason.